Modules/renderBooks.js

```diff
@@ -0,0 +1,49 @@
+import { loadLocalStorage } from './localStorage.js';
+import Book from './Books.js';
+
+export function addBook(book) {
+  Book.addBook(book);
+}
+
+function removeBook(bookId) {
+  Book.removeBook(bookId);
+}
+
+const renderBook = (book) => {
+  const { title, author } = book;
+
+  const renderContainer = document.createElement('tr');
+  const tdTitle = document.createElement('td');
```

Comment:
- [ ] as mentioned in the requirements you should Use ES6 syntax, this includes making use of arrow functions instead of the `function` keyword.